src/harness/testcases/WINNF_FT_S_FPR_testcase.py

```diff
@@ -0,0 +1,165 @@
+    grant_1['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+    grant_2 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_2['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_2['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+    grant_3 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_3['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_3['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+
+    # Creating conditionals for Cat B device
+    self.assertEqual(device_1['cbsdCategory'], 'B')
+    conditionalParameters = {
+        'cbsdCategory': device_1['cbsdCategory'],
+        'fccId': device_1['fccId'],
+        'cbsdSerialNumber': device_1['cbsdSerialNumber'],
+        'airInterface': device_1['airInterface'],
+        'installationParam': device_1['installationParam'],
+        'measCapability': device_1['measCapability']
+    }
+    del device_1['cbsdCategory']
+    del device_1['airInterface']
+    del device_1['installationParam']
+
+    # Create the actual config.
+    conditionals = {
+        'registrationData': [conditionalParameters]
+    }
+    config = {
+      'registrationRequests': [device_1, device_2, device_3],
+      'conditionalRegistrationData': conditionals,
+      'grantRecords': [grant_1, grant_2, grant_3],
+      'gwblRecords': [gwbl],
+      'fssRecords': [fss]
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_FPR_5_default_config)
+  def test_WINNF_FT_S_FPR_5(self, config_filename):
+    """[Configurable] Grant Requests from one or more CBSDs Inside an FSS-GWBL Exclusion Zone."""
+
+    config = loadConfig(config_filename)
+        
+    # Light checking of the config file
+    self.assertEqual( len(config['registrationRequests']), 3)
+    self.assertEqual( len(config['fssRecords']), 1)
+    self.assertEqual( len(config['gwblRecords']), 1)
+
+    # Load the FSS
+    self._sas_admin.InjectFss({'record': config['fssRecords']})
+
+    # Load the GWBL
+    self._sas_admin.InjectWisp(config['gwblRecords'])
+
+    # Trigger CPAS activity if GWBL > 0
+    if (len(config['gwblRecords'])):
+      self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete() 
+
+    # Register N > 0 CBSDs
+    # Check registration response
+    # The assertRegistered function does the Inject FCC ID and user ID for the registration requests
+    if ('conditionalRegistrationData' in config) and (config['conditionalRegistrationData']):
+      cbsd_ids = self.assertRegistered(config['registrationRequests'],config['conditionalRegistrationData'])
+    else:
+      cbsd_ids = self.assertRegistered(config['registrationRequests'])
+
+    # number of grants and devices should match
+    self.assertEqual(len(config['registrationRequests']), len(config['grantRecords']))
```

Comment:
Lines 145, 146, can be grouped along with the initial checking of the config file.